mods/give_initial_stuff/init.lua

```diff
@@ -1,12 +1,20 @@
-minetest.register_on_newplayer(function(player)
-	--print("on_newplayer")
+local stuff_string = minetest.setting_get("initial_stuff") or
+		"default:pick_steel, default:axe_steel, default:shovel_steel," ..
+		"default:torch 99, default:cobble 99"
+
+give_initial_stuff = {
+	stuff = stuff_string:split(",")
+}
+
+function give_initial_stuff.give(player)
 	if minetest.setting_getbool("give_initial_stuff") then
-		minetest.log("action", "Giving initial stuff to player "..player:get_player_name())
-		player:get_inventory():add_item('main', 'default:pick_steel')
-		player:get_inventory():add_item('main', 'default:torch 99')
-		player:get_inventory():add_item('main', 'default:axe_steel')
-		player:get_inventory():add_item('main', 'default:shovel_steel')
-		player:get_inventory():add_item('main', 'default:cobble 99')
+		minetest.log("action",
+				"Giving initial stuff to player " .. player:get_player_name())
+		local inv = player:get_inventory()
+		for _, item in ipairs(give_initial_stuff.stuff) do
+			inv:add_item("main", item:trim())
```

Comment:
Suggestion: Use `ItemStack(item)` instead of `item:trim()` so it will be possible to give items with custom metadata or whatever else `ItemStack` supports.